cookbooks/S20RTS_gmg.prm

```diff
@@ -0,0 +1,34 @@
+include $ASPECT_SOURCE_DIR/cookbooks/S20RTS.prm
+
+set Dimension = 3
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type = block GMG
+    set Linear solver tolerance = 1.e-7
+    set Number of cheap Stokes solver steps = 0
+  end
+end
+
+subsection Material model
+  set Material averaging =  harmonic average
+end
+
+subsection Postprocess
+  subsection Visualization
+    set Output mesh velocity = true
```

Comment:
I do not understand this output, shouldnt this always be 0 because this cookbook does not use a free surface?